lv_conf_template.h

```diff
@@ -600,6 +600,9 @@
 #define LV_FONT_SOURCE_HAN_SANS_SC_14_CJK   0  /**< 1000 most common CJK radicals */
 #define LV_FONT_SOURCE_HAN_SANS_SC_16_CJK   0  /**< 1000 most common CJK radicals */
 
+/** Enables/disables usage of static bitmaps */
+#define LV_USE_STATIC_BITMAP 0 
```

Comment:
We can decide it font by font. Let's add a `static_bitmap` bit to `lv_font_t` which is set by default, so that we can decide for each font if the bitmap is static.

Reply:
I agree, but at the same time, `LV_USE_STATIC_BITMAP` disables [this](https://github.com/lvgl/lvgl/pull/8156/files#diff-980ca683736d1a7b4999cdc05f82d431c4aef098d9a85f075e357fa63b7ca427L597) part of the code which improves the execution time by skipping the allocation of the `draw_buf`, which is not necessary when using the static bitmaps.

With `bool lv_font_has_static_bitmap(font);` it would require an additional if to be checked at runtime.

If we want the maximum performance, the define would be the best option.

Reply:
Normally only a few reallocations are happening here while buffer reaches a large enough size. 

I'd rather make this part allocate some extra, e.g. round the size to next 128 byte boundary. This way it should be typically 1 allocations per text, which shouldn't matter.